src/modules/positions/ui/positions-list.tsx

```diff
@@ -0,0 +1,73 @@
+import { Position } from "cv-graphql";
+import { t } from "i18next";
+import ListTemplate from "@/components/list-lemplate/list-template";
+import { Action } from "@/components/list-lemplate/actions-menu";
+import { ColumnConfig } from "@/components/list-lemplate/table-template";
+import { useConfirm } from "@/components/confirm-dialog/";
+import { usePositionCreate, usePositionDelete, usePositionsQuery, usePositionUpdate } from "../api";
```

Comment:
since usePositionsQuery is now part of common API (also needed in profile), probably should replace this with the global one